### ISSUE TITLE
Missing styling for tabs

### ISSUE THREAD

**Issue by lbolla** [monkeyjunglejuice/matrix-emacs-theme#6]
Tabs are unstyled so they don't look very nice.

doom-themes have a very nice way of styling them that this theme can take inspiration from.

**Comment by monkeyjunglejuice** [comment]
Good idea, thank you. I never used them actually, so they weren't on my list.

**Comment by lbolla** [comment]
I've submitted a PR: https://github.com/monkeyjunglejuice/matrix-emacs-theme/pull/7

**Comment by monkeyjunglejuice** [comment]
The Matrix theme has a pretty tab-bar and tab-line now :)
